cmd/kops/rollingupdatecluster.go

```diff
@@ -49,7 +50,7 @@ var (
 	Note: terraform users will need run the following commands all from the same directory "kops update cluster --target=terraform" then "terraform plan" then "terraform apply"
 	prior to running "kops rolling-update cluster"
 
-	Use export KOPS_FEATURE_FLAGS="+DrainAndValidateRollingUpdate" to use beta code that drains the nodes
+	Use ` + pretty.Bash("export KOPS_FEATURE_FLAGS=\"+DrainAndValidateRollingUpdate\"") + ` to use beta code that drains the nodes
```

Comment:
Tip: I _think_ you can use backticks here to avoid escaping the `"` quote.  I'm not going to comment on whether that would be more or less confusing given the whole point is to put the output in backticks :-)